core/src/main/java/org/mskcc/cbio/portal/dao/DaoMutation.java

```diff
@@ -140,16 +140,17 @@ public static void createMutationCountClinicalData(GeneticProfile geneticProfile
             // do not include germline and fusions (msk internal) when counting
             // mutations
             pstmt = con.prepareStatement(
-                    "SELECT `SAMPLE_ID`, COUNT(DISTINCT mutation_event.`CHR`, mutation_event.`START_POSITION`, " +
+                    "SELECT sample_profile.`SAMPLE_ID`, COUNT(DISTINCT mutation_event.`CHR`, mutation_event.`START_POSITION`, " +
                     "mutation_event.`END_POSITION`, mutation_event.`REFERENCE_ALLELE`, mutation_event.`TUMOR_SEQ_ALLELE`) AS MUTATION_COUNT " +
-                    "FROM `mutation` , `genetic_profile`, `mutation_event` " +
-                    "WHERE mutation.`GENETIC_PROFILE_ID` = genetic_profile.`GENETIC_PROFILE_ID` " +
-                    "AND mutation.`MUTATION_EVENT_ID` = mutation_event.`MUTATION_EVENT_ID` " +
-                    "AND mutation.`MUTATION_STATUS` <> 'GERMLINE' " +
+                    "FROM `sample_profile` " +
+                    "LEFT JOIN mutation ON mutation.`SAMPLE_ID` = sample_profile.`SAMPLE_ID`" +
+                    "LEFT JOIN mutation_event ON mutation.`MUTATION_EVENT_ID` = mutation_event.`MUTATION_EVENT_ID`" +
```

Comment:
@ersinciftci maybe good to add a comment to this query - just to clarify that we're counting 0 when the sample is profiled. No count in MUTATION_COUNT when it hasn't been

Reply:
@inodb done